example/app.js

```diff
@@ -81,6 +81,10 @@ search.addWidget(
     templates: {
       header: '<div class="panel-heading">Price ranges</div>',
       item: require('./templates/and.html')
+    },
+    transformData: function(data) {
+      data.name = data.name.replace(/(\d+) - (\d+)/, '$$$1 - $$$2').replace(/> (\d+)/, '> $$$1');
```

Comment:
@Jerskouille you can clone `data` here and mutate name. It would be fine.
@vvo it doesn't completely copy the resulting data... and not at all for the hierarchical facets.